libraries/entities/src/EntityTree.cpp

```diff
@@ -1807,6 +1814,12 @@ int EntityTree::processEditPacketData(ReceivedMessage& message, const unsigned c
         return 0;
     }
 
+    if (wantAuditEditLogging() && !entitiesAuditLogProcessor.isProcessorRunning()) {
+        entitiesAuditLogProcessor.startAuditLogProcessor();
+    }
+
+    qDebug() << "PROCESSING" << wantAuditEditLogging() << " - " << entitiesAuditLogProcessor.isProcessorRunning();
```

Comment:
```suggestion
    qCDebug(entities) << "PROCESSING" << wantAuditEditLogging() << "-" << entitiesAuditLogProcessor.isProcessorRunning();
```
Spaces are automatically included between items.